sdk/src/main/java/co/omise/android/ThreeDS2ServiceWrapper.kt

```diff
@@ -37,12 +37,20 @@ internal class ThreeDS2ServiceWrapper(
                 .rootPublicKey(BuildConfig.DS_PUBLIC_KEY)
                 .build()
             val configParameters = ConfigurationBuilder()
-                .license(BuildConfig.NETCETERA_LICENSE_KEY)
+                .apiKey(BuildConfig.NETCETERA_API_KEY)
                 .configureScheme(schemeConfig)
                 .build()
             val locale = getLocale()
-            threeDS2Service.initialize(context, configParameters, locale, uiCustomization)
-            continuation.resume(Result.success(Unit))
+            threeDS2Service.initialize(context, configParameters, locale, uiCustomizationMap,object :
```

Comment:
```suggestion
            threeDS2Service.initialize(context, configParameters, locale, uiCustomizationMap, object :
```

Reply:
@AnasNaouchi please make a ticket for implementing and enforcing some kind of linter for these kinds of things?